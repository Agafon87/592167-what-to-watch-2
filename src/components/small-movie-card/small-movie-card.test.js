import React from "react";
import renderer from "react-test-renderer";
import SmallMovieCard from "../small-movie-card/small-movie-card.jsx";

it(`SmallMovieCard correctly renders after test`, () => {
  const tree = renderer
    .create(<SmallMovieCard filmCard={{}}/>)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
