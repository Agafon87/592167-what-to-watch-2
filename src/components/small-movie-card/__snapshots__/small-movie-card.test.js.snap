// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SmallMovieCard component renders correctly 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
>
  <a
    href="/films/4"
    onClick={[Function]}
  />
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="/films/4"
      onClick={[Function]}
    >
      Авиатор
    </a>
  </h3>
</article>
`;
